scalafmt.conf

```diff
@@ -0,0 +1,2 @@
+runner.dialect = scala213
```

Comment:
As in, Scala 2.13 (the last major version of Scala 2.x; there's not gonna ever be any Scala 2.14, just Scala 3.x)